### ISSUE TITLE
New feature on Material Tracker makes it difficult to use

### ISSUE THREAD

**Issue by octavebits** [Aceship/AN-EN-Tags#26]
Thanks for making toolbox, super helpful.

Recently implemented feature on Material Tracker automatically coverts lower tier items to it's corresponding higher tier craft. Perhaps this directly demonstrates how many units are needed to craft higher tier and shows it realtime, but it's inconvenient and disruptive when using Material Tracker to keep inventory.

The old version served it's purpose well. It provided visibility so that users didn't have to go to depot every time they wanted to check inventory. And provided awareness by calculating and showing how much more of an item or lower tier parts would be needed.

By automatically converting lower tiers, we can no longer confirm at glance how much inventory we have, and thus by extension, lowering confidence in the calculated results.

Basically, it becomes a simulator instead of inventory tracking/planner.

**Comment by Aceship** [comment]
Thanks for the feedback, the automatic convert should be only happen if you use the update button now, adding material no longer automatically convert to higher one

page should be updated in few minutes after this

**Comment by octavebits** [comment]
Thanks so much! =D

**Comment by Aceship** [comment]
no prob 👍
